features/mobile/not_safe_for_work.feature

```diff
@@ -0,0 +1,74 @@
+@javascript @mobile
+Feature: Not safe for work
+  Background:
+    Given following users exist:
+      | username    | email             |
+      | pr0n king   | tommy@pr0n.xxx    |
```

Comment:
[Am I not](https://github.com/diaspora/diaspora/pull/6245/files#diff-3bdf1f0e536df0353401430184957034R44) ?
Besides this, this whole file as been adapted from the desktop one. So I woudn't be able to justify every choice. I just know that it works, that's all.